docs/src/folderstructure.md

```diff
@@ -2,7 +2,7 @@
 
 [ClimaLand](https://github.com/CliMA/ClimaLand.jl) home directory has 6 main folders:
 
-- docs: contains files to generate [the documentation website](clima.github.io/ClimaLand.jl/).
+- docs: contains files to generate [the documentation website](https://clima.github.io/ClimaLand.jl/dev/).
```

Comment:
I think now that we fixed stable docs, that's what we should use.
https://clima.github.io/ClimaLand.jl/stable/

can you edit the README.md in this PR to update the badge from dev to stable as well?